contracts/utils/addresses.js

```diff
@@ -245,7 +245,15 @@ addresses.mainnet.CurveCVXPool = "0xB576491F1E6e5E62f1d8F26062Ee822B40B0E0d4";
 addresses.mainnet.SSV = "0x9D65fF81a3c488d585bBfb0Bfe3c7707c7917f54";
 addresses.mainnet.SSVNetwork = "0xDD9BC35aE942eF0cFa76930954a156B3fF30a4E1";
 addresses.mainnet.beaconChainDepositContract =
-  "0x00000000219ab540356cbb839cbe05303d7705fa";
+  "0x00000000219ab540356cBB839Cbe05303d7705Fa";
+
+// Native Staking Strategy
+addresses.mainnet.nativeStakingSSVStrategyProxy =
+  "0xdF58F78cebbb2A60740eD2f86cDf0545a485102F";
+addresses.mainnet.nativeStakingFeeAccumulatorProxy =
```

Comment:
The nativeStakingFeeAccumulator and SSV Strategy proxy seem to not be deployed to the above addresses or anywhere for the moment

Reply:
I will just comment them out for the moment

Reply:
I think we can remove these. I had forgotten we can get the address from the deployment files.